includes/Admin/Crop_Script.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Class Crop_Script.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Admin;
+
+use Google\Web_Stories\Assets;
+use Google\Web_Stories\Infrastructure\Conditional;
+use Google\Web_Stories\Infrastructure\Registerable;
+use Google\Web_Stories\Infrastructure\Service;
+use Google\Web_Stories\Traits\Screen;
+
+/**
+ * Class Crop Script
+ *
+ * @package Google\Web_Stories
+ */
+class Crop_Script implements Conditional, Service, Registerable {
+	use Screen;
+
+	/**
+	 * Web Stories editor script handle.
+	 *
+	 * @var string
+	 */
+	const SCRIPT_HANDLE = 'imgareaselect';
+
+	/**
+	 * Assets instance.
+	 *
+	 * @var Assets Assets instance.
+	 */
+	private $assets;
+
+	/**
+	 * Crop Script constructor.
+	 *
+	 * @since 1.10.0
+	 *
+	 * @param Assets $assets Assets instance.
+	 */
+	public function __construct( Assets $assets ) {
+		$this->assets = $assets;
+	}
+
+	/**
+	 * Check whether the conditional object is currently needed.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @return bool Whether the conditional object is needed.
+	 */
+	public static function is_needed(): bool {
+		return is_admin() && ! wp_doing_ajax();
+	}
+
+	/**
+	 * Runs on instantiation.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @return void
+	 */
+	public function register() {
+		add_filter( 'script_loader_tag', [ $this, 'script_loader_tag' ], 10, 3 );
+	}
```

Comment:
I looked at unregistering the script. However, the using this filter, means I don't have to register dependencies and means that is_edit_screen is usable.